### PR TITLE
build: resolve comments on flat modules PR

### PR DESCRIPTION
**Pull request by jasonaden**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -54,33 +54,24 @@ for ARG in "$@"; do
...
 }
 
-containsElement () {
-  local e
-  for e in "${@:2}"; do
-    [[ "$e" == "$1" ]] && return 0;
-  done
-  return 1
+#######################################
+# Downsamples ES2015 to ESM/ES5
```
Comment (IgorMinar) [inline build.sh]: we've been using "down-level" terminology in the past. can you please update?

File: build.sh [hunk 2]
```diff
@@ -187,11 +178,15 @@ do
 
   rm -rf ${DESTDIR}
 
-  echo "======      [${PACKAGE}]: COMPILING: ${TSC} --skipImportRename -p ${SRCDIR}/tsconfig-build.json"
+  # When .babelrc file exists, the dist package will have ES2015 sources, ESM/ES5, and UMD bundles. Because of a bug
+  # preventing the Compiler package from running through this pipeline, we have to manually check for the Compiler
```
Comment (IgorMinar) [inline build.sh]: `the Compiler package` -> `@angular/compiler package`

File: build.sh [hunk 3]
```diff
@@ -187,11 +178,15 @@ do
 
   rm -rf ${DESTDIR}
 
-  echo "======      [${PACKAGE}]: COMPILING: ${TSC} --skipImportRename -p ${SRCDIR}/tsconfig-build.json"
+  # When .babelrc file exists, the dist package will have ES2015 sources, ESM/ES5, and UMD bundles. Because of a bug
```
Comment (IgorMinar) [inline build.sh]: can you add a reference to that bug please?

**Review by IgorMinar — APPROVED** [review]

File: build.sh (hunk 1 above)
Comment (jasonaden) [inline build.sh]: done

File: build.sh (hunk 2 above)
Comment (jasonaden) [inline build.sh]: Done

**Pull request merged**